test/extended/networking/acl_audit_log.go

```diff
@@ -109,9 +109,7 @@ func testACLLogging(f *e2e.Framework, oc *exutil.CLI, ns []string, auditOut stri
 	}
 
 	By("making \"default deny\" and \"allow from same namespace\" network policies")
-	// make two network policies, a default deny all and allow from same namespace in ns "acl-logging-on"
-	_, err = makeDenyAllPolicy(f, ns[0])
-	expectNoError(err)
+	// a make network policy that allows ingress only from the same ns "acl-logging-on"
 	_, err = makeFromSameNSPolicy(f, ns[0])
```

Comment:
the `By()` message is now wrong.
(Just get rid of the comment put it all in the `By()` message; then the explanation is visible both to people reading the code and to people reading test logs)

Reply:
I will clean this up in a followup